libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BeginSkill.schema

```diff
@@ -76,6 +76,26 @@
             "title": "Activity",
             "description": "The activity to send to the skill."
         },
+        "deliveryMode": {
+            "$ref": "schema:#/definitions/stringExpression",
+            "title": "Delivery mode",
+            "description": "Defines the expected delivery response type for the call to the skill.",
+            "oneOf": [
+                {
+                    "type": "string",
+                    "title": "Delivery mode",
+                    "description": "Delivery mode type.",
+                    "enum": [
+                        "normal",
+                        "expectReplies"
+                    ],
+                    "default": ""
```

Comment:
I'm not exactly sure where to put this feedback, but shouldn't this property have a default? (in the schema and the code).

And then, if that's true, what should that value be, there has been discussion that it should be expectReplies, as immediate reply is by far the more common semantic.

Further to that point, shouldn't the _description_ say that the only time you would want to not use expectReplies is when your skill wants to do update, delete and reactions and your channel supports it. Having just a stand alone user visible property "delivery mode / expect replies" seems fairly cryptic.

Reply:
Based on the spec, '' is the same as 'normal', which is the default defined in the spec.

I did at first set a default of 'expectReplies', then realized this will change existing behavior: so would be a breaking change.

I considered adding more descriptive text, describing 'expectReplies', in this popup:

![image](https://user-images.githubusercontent.com/11055362/96759802-5efeb780-138d-11eb-8418-378c64aba02d.png)

The entire feature is somewhat cryptic, especially considering that if the context.Activity.Type is 'invoke', this property is ignored.  Or, if they Activity being sent has a DeliveryMode, this property will be ignored.  It is more like a 'SuggestedDeliveryMode', with multiple other fields taking precedence otherwise, we break backward compat behavior. 

I was also going to add some tests for this, but waiting for this PR to be merged: https://github.com/microsoft/botbuilder-dotnet/pull/4816  so i can update it.

Reply:
I am moving the underlying issue to R12, we can leave this PR active (will change it to draft though). 

I'd rather have a call with the UX team for composer to see how they want this surfaced, I agree that it is super cryptic and we cannot default it expectReplies because that will be a change in behavior.